public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -310,10 +310,13 @@ function getVizPanelTransformations(vizPanel: VizPanel): TransformationKind[] {
           id: transformation.filter?.id ?? '',
           options: transformation.filter?.options ?? {},
         },
-        topic: transformation.topic,
         options: transformation.options,
       };
 
+      if (transformation.topic !== undefined) {
+        transformationSpec.topic = transformation.topic;
```

Comment:
is topic persisted at all? should it be part of the transformtion filter?

Reply:
Yes, it's persisted, and this matches the current shape (not a part of transformation filters). This is what I see in dashboard JSON
```
  {
          "filter": {
            "id": "byRefId",
            "options": ""
          },
          "id": "limit",
          "options": {},
          "topic": "annotations"
}
```